build-logic/src/main/groovy/org.apache.groovy-performance.gradle

```diff
@@ -70,7 +70,7 @@ tasks.named('compileJmhGroovy') {
 
 tasks.register('displayJmhResults') {
     doLast {
-        println file("$buildDir/results/jmh/results.txt").text
+        println layout.buildDirectory.file("results/jmh/results.txt").get().asFile
```

Comment:
This currently actually prints out the file, not the name as proposed, so the .text is still needed.

Reply:
Good catch!